### PR TITLE
fix: hide dependency warning if the user can run kubectl kustomize

### PR DESCRIPTION
**Pull request by Toxer10**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -26,7 +26,12 @@ export const checkMissingDependencies = (dependencies: Array<string>): Array<str
         },
       });
       return false;
-    } catch (e) {
+    } catch (e: any) {
+      // If kubectl kustomize was ran and threw an exception that it could not find any kustomization file - should return false
+      if (e.message.includes("unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization'")) {
```
Comment (devcatalin) [inline src/utils/index.ts:31]: I think you can remove this and run `kubectl kustomize --help` instead so you get an error only if it's not recognized

**Review by devcatalin — CHANGES_REQUESTED** [review]

File: src/utils/index.ts (hunk 1 above)
Comment (Toxer10) [reply src/utils/index.ts:31]: Thanks for the advice!

Comment (Toxer10) [reply src/utils/index.ts]: Done

**Review by devcatalin — APPROVED** [review]
lgtm!

**Pull request merged**